README.md

```diff
@@ -621,13 +621,38 @@ npm install prosemirror-utils
    ```
 
 
- * **`cloneRowAt`**`(cloneRowIndex: number) → fn(tr: Transaction) → Transaction`\
-   Returns a new transaction that adds a new row after `cloneRowIndex`, cloning the row attributes at `cloneRowIndex`.
+ * **`copyRow`**`(insertNewRowIndex: number, rowToBeClonedIndex: number, options: ?CopyRowOptions) → fn(tr: Transaction) → Transaction`\
+   Returns a new transaction that adds a new row at index `insertNewRowIndex`.
+   Copying the cells from any row using `rowToBeClonedIndex`,
+   you can configure how that copy should working by the options:
+
+   ```javascript
+   options = {
+     /**
+      * if you want to keep the colspan from the cloned row,
+      * by default, it will split the cells.
+      *
+      */
+     keepColspan: false,
+     /**
+      * To keep the deprecated behavior from `cloneRowAt`
+      * that clones the row and increases the rowspan from the previous rows.
+      */
+     expandRowspanFromClonedRow: false
```

Comment:
how is it going to work if `rowToBeClonedIndex` is not the previous row? 
Could we always clone the previous row and remove `rowToBeClonedIndex`?

Reply:
Also, you can't keep colSpans if you insert a row in the middle of a `rowSpanned` row, because you'd have too many cells in the new row.

Reply:
> how is it going to work if rowToBeClonedIndex is not the previous row?

This is a problem, good catch, we can add a check, since this is deprecated. 

> Could we always clone the previous row and remove rowToBeClonedIndex?

This will limit that function, for example:

```javascript
    tr = copyRow(7, 4, {
      expandRowspanFromClonedRow: false,
      keepColspan: true,
      getNewCell(cell) {
        return cell.type.createAndFill({ background: cell.attrs.background });
      },
    })(state.tr);
```
We will have this:
![Screen Recording 2019-08-05 at 11 32 am](https://user-images.githubusercontent.com/3452187/62432770-cfafb900-b774-11e9-8fba-5950459a0141.gif)


> Also, you can't keep colSpans if you insert a row in the middle of a rowSpanned row, because you'd have too many cells in the new row.

What do you mean? This is well tested, check this out:

![Screen Recording 2019-08-05 at 11 38 am](https://user-images.githubusercontent.com/3452187/62432937-9cb9f500-b775-11e9-9cd4-1c19185be69d.gif)

Reply:
> how is it going to work if rowToBeClonedIndex is not the previous row?

I added a check for it, it will throw a error on that case.

Reply:
why do we need this param if it only works with previous rows?

Reply:
Only to keep the `cloneRowAt` working without any breaking change